kloppy/tests/test_sportec.py

```diff
@@ -69,9 +77,48 @@ def test_correct_deserialization(self, event_data: Path, meta_data: Path):
 
         assert dataset.events[0].coordinates == Point(56.41, 68.0)
 
-    def test_correct_normalized_deserialization(
+    def test_correct_normalized_event_data_deserialization(
         self, event_data: Path, meta_data: Path
     ):
-        dataset = sportec.load(event_data=event_data, meta_data=meta_data)
+        dataset = sportec.load_event(
+            event_data=event_data, meta_data=meta_data
+        )
 
         assert dataset.events[0].coordinates == Point(0.5640999999999999, 1)
+
+    def test_load_tracking_data(self, raw_data: Path, meta_data: Path):
+        dataset = sportec.load_tracking(
+            raw_data=raw_data, meta_data=meta_data, coordinates="sportec"
+        )
+
+        home_team, away_team = dataset.metadata.teams
+
+        assert dataset.frames[0].ball_owning_team == away_team
+        assert dataset.frames[0].ball_state == BallState.DEAD
+        assert dataset.frames[0].ball_coordinates == Point3D(
+            x=2.69, y=0.26, z=0.06
+        )
+
+        assert dataset.frames[1].ball_owning_team == home_team
+        assert dataset.frames[1].ball_state == BallState.ALIVE
+
+        player_lilian = away_team.get_player_by_id("DFL-OBJ-002G3I")
+        player_data = dataset.frames[0].players_data[player_lilian]
+
+        assert player_data.coordinates == Point(x=0.35, y=-25.26)
+
+        # We don't load distance right now as it doesn't
```

Comment:
There is also an `Acceleration` attribute in the Sportec data, is there a possibility to add this to the dataset?

Reply:
Jup. Would like to implement this in a different ticket to first check other providers too.